draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1289,27 +1296,32 @@ The following procedure implements hash\_to\_base.
 hash_to_base(msg, ctr)
 
 Parameters:
-- H, a cryptographic hash function producing b bits.
+- DST, a domain separation tag (see discussion above).
+- H, a cryptographic hash function.
 - F, a finite field of characteristic p and order q = p^m.
-- W = ceil((ceil(log2(p)) + k) / b), where k is the security
+- L = ceil((ceil(log2(p)) + k) / 8), where k is the security
   parameter of the cryptosystem (e.g., k = 128).
+- HKDF-Extract-H is the HKDF-Extract function of RFC5869
```

Comment:
I find this syntax a little confusing. Is the idea that the H in `HKDF-Extract-H` should be expanded to SHA2, etc, in each case?

I suggest we stick to the same notation used in the original draft, and others like the TLS 1.3 draft. So, just use "HKDF-Extract" and specify under that the hash function used is given by the ciphersuite?

Reply:
What if we just called it "Extract"?

```
- Extract is the HKDF-Extract function of RFC5869 instantiated with the hash function H.
```

Is that clearer?

Reply:
Maybe:

```
 - HKDF-Expand and HKDF-Extract are as defined in {{rfc5869}}, instantiated with the hash function H
```
Again, keeping it closer to notation used elsewhere.

Reply:
I preferred the -H notation since it made clear that H determined Extract, though the expansion issue is a valid concern. I'm fine with the proposal!